docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+
+- Edna can create and read customer information
+- Edna can modify customer information.
+- Edna can overwrite or replace credit card number, expiration, and CVV information.
+
+> In the Contoso Webstore, the user is automatically as the **Edna** user for testing the capabilities of the deployed environment.
+
+### Contoso Webstore - Azure pricing sample calculation
+
+The solution cost sample has a monthly fee structure and a use per hour to
+consider when sizing the solution. This example deployment **estimated** cost using the [Azure costing calculator](https://azure.microsoft.com/en-us/pricing/calculator/). The solution 
+consist of the following items:
+
+| **Service type** | **Custom name** | **Region** | **Description** | **Estimated Cost** | 
+| ----------------- | --------------- | ----------- | -------------- | ----------------- | 
+| Virtual Machines | Virtual Machines | South Central US | 1 Standard virtual machine(s), 1 Standard virtual machine(s), A2 v2 (2 cores, 4 GB RAM, 20 GB disk) size: 744 hours | $101.18 | 
+| App Service | App Service | South Central US | 1 instance(s), 744 hours, size: P1, premium tier, 0 SNI connection(s), 0 IP connection(s) | $223.20 | 
+| IP Addresses | IP Addresses | East US | arm type, 2 public IP Address(es) x 744 hours | $5.95 | 
+| SQL Database | SQL Database | East US | 1 standard database(s) x 1 months, size: s0 | $15.03 | 
+| Storage | Storage | East US | 5/GB storage: Block blob type, Basic tier, LRS redundancy | $0.10 | 
+| Storage | Storage | East US | 1 GB storage Table and Queue type. Basic tier, LRS redundancy, 1 x100,000 transactions | $0.07 | 
+| Storage | Storage | East US | standard-s4 Disk type with 1 Managed disks | $0.77 | 
+| Application Insights | Application Insights | East US | basic tier in us-east region with 2 GBs and 0 multi-step web test(s). | $2.30 | 
+| Log Analytics | Log Analytics | East US | 1 GB(s), standalone tier | $2.30 | 
+| Security Center | Security Center | East US |  | $15.00 | 
+| Key Vault | Key Vault | East US | 1000 operations, 0 certificate renewals, 0 HSM keys in the us-east region | $0.03 | 
+| Azure Active Directory | Azure Active Directory | East US | free tier, per-user MFA billing model, 10 MFA user(s), 25001-100000 directory objects, 0 hours | $14.00 | 
+| Application Gateway | Application Gateway | East US | 1 instance(s) x 1 months, 1 GB data processed, outbound transfers:: 5 GB | $93.74 | 
+| | | | | **Monthly Total $473.67** |
+| | | | | **Annual Total  $5,684.04** |
+ 
+Disclaimer 
+All prices shown are in US Dollars ($). This estimate was created in April 2017. [EDIT: ADD GUIDANCE]
+
+This solution used the following Azure services (details to the deployment
+architecture are located in the [Deployment Architecture](#deployment-architecture) section).
+
+>- Application Gateway
```

Comment:
This list isn't particularly helpful unless there is more information about the purpose of each component.